packaging/rpm/microshift-airgap.spec

```
@@ -0,0 +1,31 @@
+Name: microshift
+Version: %{version}
+Release: %{release}%{dist}
+# this can be %{timestamp}.git%{short_hash} later for continous main builds
+Summary: Create custom container storage
+License: ASL 2.0
+URL: https://github.com/redhat-et/microshift
+
+%description
+This rpm creates a custom RO container storage and pull images for disconnected scenarios.
+
+%prep
+
+%build
+
+
+%install
+mkdir -p /var/lib/microshift/airgap-images
+sudo sed -i '/^additionalimagestores =*/a "/var/lib/microshift/airgap-images",' /etc/containers/storage.conf
```

```suggestion
sudo sed -i '/^additionalimagestores =*/a "/var/lib/microshift/airgap-images",' /etc/containers/storage.conf
```
this bit should go in %post ^ right before the systemctl restart